tutorials/users/ide/ide.md

```diff
@@ -1 +1,9 @@
 # IDE
+
+Jupyter is a web-based development environment well suited for arranging workflows in 
```

Comment:
Let's talk about JupyterLab specifically (not just Jupyter)

Reply:
Changed to keep the topic about JupyterLab